Devfile_Cert_Criteria.md

```diff
@@ -0,0 +1,84 @@
+# Certified Stack Hub
+
+This document describes the criteria a Devfile application stack must implement to be considered a certified application stack.
+
+## Certification Criteria
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this
+document are to be interpreted as described in [BCP
+14](https://tools.ietf.org/html/bcp14)
+[[RFC2119](https://tools.ietf.org/html/rfc2119)]
+[[RFC8174](https://tools.ietf.org/html/rfc8174)] when, and only when, they
+appear in all capitals, as shown here.
+
+### Stack structure
+
+* A certified Application Stack MUST also be an [Application Stack](https://github.com/IBM/cp4apps-hub/tree/devfile_cert#application-stacks), having the necessary files needed to fulfill core scenarios by the architect (Champ) and the developer (Jane).
```

Comment:
We should add a link to the devfile information https://odo.dev/file-reference/ that describes what is in the devfile.